src/metricscollector/collector/apppoller_test.go

```diff
@@ -154,38 +191,39 @@ var _ = Describe("Apppoller", func() {
 				}
 			})
 
-			It("saves successful results to database", func() {
-				Expect(database.SaveMetricCallCount()).To(Equal(2))
+			It("saves successful results to database and logs the errors", func() {
+				fclock.Increment(TestPollInterval * time.Second)
+				Eventually(database.SaveMetricCallCount).Should(Equal(1))
+
+				fclock.Increment(TestPollInterval * time.Second)
+				Consistently(database.SaveMetricCallCount).Should(Equal(1))
+				Eventually(buffer).Should(gbytes.Say("apppoller test error"))
```

Comment:
This is unneeded since you are tracking save call counts and we have proven that it saves on regular intervals.